django_freeradius/base/admin.py

```diff
@@ -206,7 +206,26 @@ def get_readonly_fields(self, request, obj=None):
 
 
 class AbstractRadiusProfileAdmin(TimeStampedEditableAdmin):
-    pass
+    list_display = ['name', 'get_daily_session_limit', 'get_daily_bandwidth_limit', 'max_all_time_limit',
+                    'created', 'modified']
+
+    def get_daily_session_limit(self, obj):
+        # returns daily session limit in hours
+        if obj.daily_session_limit:
+            return round(obj.daily_session_limit / 3600, 2)
+    get_daily_session_limit.short_description = 'daily session limit(hours)'
```

Comment:
Space after limit and before the parenthesis. `daily session limit (hours)`

Reply:
it should be `_('daily session limit (hours)')`, human readable strings needs to be internationalization ready for the future

Reply:
@nemesisdesign is right. It slipped out of my mind yesterday.